app/models/api/v1/offering.rb

```diff
@@ -129,6 +129,13 @@ def initialize(offering, protocol, host_with_port, current_user)
         self.external_reports << report_attributes(report, offering, protocol, host_with_port)
       end
     end
+    if additional_external_report_id
+      additional_report = ExternalReport.find_by_id(additional_external_report_id)
+      if additional_report
+        self.external_reports << report_attributes(additional_report, offering, protocol, host_with_port)
```

Comment:
The lines from 133 to 135 are not covered by tests. Can you update the tests to cover these?

There is an empty spec at models/api/v1/offering_spec.rb that you could fill out.
But since it is empty I'd guess this code is mostly covered by the controllers/api/v1/offering_controller_spec.rb which does have a "when offering has an external report" spec.

Reply:
Just did it. I've used `controllers/api/v1/offering_controller_spec.rb` and updated one other test to use `external_reports` instead of deprecated `external_report`.

Reply:
Looks good.